<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SellsPage</class>
 <widget class="QWidget" name="SellsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UrgeMarket Sells</string>
  </property>
  <layout class="QVBoxLayout" name="topLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>My Listings:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="listingsTableWidget">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Id</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Expires</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Price</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Title</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Status</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Buy Requests:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="buysTableWidget">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Date</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Status</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Buyer</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Item</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Item Id</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Request Id</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="createButton">
       <property name="text">
        <string>Create &amp;Listing</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>&amp;Cancel Listing</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="acceptButton">
       <property name="text">
        <string>&amp;Accept Buy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rejectButton">
       <property name="text">
        <string>&amp;Reject Buy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyAddressButton">
       <property name="text">
        <string>Copy &amp;Buyer Address</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="cancelEscrowButton">
       <property name="text">
        <string>Ca&amp;ncel Escrow</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="requestPaymentButton">
       <property name="text">
        <string>Request &amp;Payment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refundButton">
       <property name="text">
        <string>Ref&amp;und</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Note: Escrow may only be cancelled prior to the buyer completing &amp;quot;escrow lock&amp;quot;, which broadcasts the escrow transaction to the network.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
